.gitignore

```diff
@@ -2,6 +2,10 @@
 .Rhistory
 .Rapp.history
 
+packages/
+
+inst/lib/

```

Comment:
@PavelBal This is new. Those dll's are not in github anymore. They need to be downloaded by script